src/banner/types.js.flow

```diff
@@ -33,6 +33,18 @@ export type ArtworkContentT = {|
   type?: ArtworkTypeT,
 |};
 
+export type OverridesT = {
```

Comment:
Can you please maintain the exact pipes? https://flow.org/en/docs/types/objects/#toc-exact-object-types

Reply:
:thumbsup: updated